### DISCUSSION TITLE
Livewire page component layout view not found

### DISCUSSION THREAD

**Discussion by cntabana** [filamentphp/filament#7554] [General]
After updating Composer this morning, I encountered the error message "**Livewire page component layout view not found.**" I've been attempting to troubleshoot the issue, but I haven't been successful so far.

I made changes to the Livewire configuration in config/livewire.php, setting 

```
 'layout' => 'components.layouts.app',
 'layout' => 'components.layouts.base',
 'layout' => 'layouts.app',
 'layout' => 'layouts.base',
```

 but unfortunately, that didn't resolve the problem. I also tried clearing the cache using php artisan view:clear, but the issue persists.

It's worth noting that this issue seems to affect Livewire components that are located outside of panels. I even attempted to create a new Livewire component using **php artisan make:livewire** and accessed it using **@livewire** in my Blade view, but I encountered the same error.

I'm looking for guidance on how to resolve this situation. Your assistance would be greatly appreciated.

**Comment by danharrin** [comment]
This isn't Filament related, please ask in the Livewire repo

**Comment by cntabana** [comment]
I think but I attempted to follow the steps outlined in this documentation: https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component#setting-up-the-livewire-component. Previously, it was functioning correctly, but as of this morning, it's no longer working.

**Comment by eorenstein1** [comment]
I'm having the same issue after upgrading to Livewire 3.0.0-beta.8. Any resource page with a widget gives me this error.

**Reply by nikcani** [reply]
exactly, a possible workaround right now is to use 3.0.0-beta.7

**Comment by demobiel** [comment]
Here is a bug report on the livewire side: https://github.com/livewire/livewire/discussions/6281

They mention another possible workaround: create an empty blade file.

**Reply by demobiel** [reply]
which doesn't work, so indeed at the moment stay on the version 3.0.0-beta.7
